frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/doclet/FrankMethod.java

```diff
@@ -17,6 +17,8 @@
 package nl.nn.adapterframework.frankdoc.doclet;
 
 public interface FrankMethod extends FrankProgramElement {
+	public static final String JAVADOC_DEFAULT_VALUE_TAG = "@frankdoc.default";
```

Comment:
Kan deze tag niet '@default' worden?

Reply:
Ik heb ergens gelezen dat dat niet verstandig zou zijn. Als Eclipse, Maven of de Java taal het tag "default" in de toekomst maakt, dan gaat dat conflicteren als wij tag "default" hebben. Er wordt aangeraden om een prefix aan de tag naam toe te voegen.

Reply:
Met ergens bedoel ik deze StackOverflow post: https://stackoverflow.com/questions/2677564/how-to-create-custom-javadoc-tags. Ik doel op de bijdrage van "knownasilya" en dan zijn alinea die begint met: "Notice that the custom tag has the format".

Reply:
laten we er dan `@ff.default` van maken.

Reply:
Gedaan.